src/content/docs/cheerpj-jnlp-runner/04-guides/private-hosting-chrome.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Private hosting (Chrome)
+description: Deploy the JNLP Runner chrome extension via Group Policy (Windows).
+---
+
+If your organisation requires to use our extension in multiple devices, we have amazing news for you! Our JNLP Runner extension is available for private hosting and deployment via Group Policy. This way, you can deploy our extension for multiple users under your organisation simultaneously and administrate it from a single place. This page will take you step by step on how to do this.
+
+Requirements:
+
+- The appropriate IT permissions at your organisation
+- The JNLP Runner extension CRX file.
+
+## The CRX File and Download URL
```

Comment:
Given it's called `DownloadURL` in Google's API, I'd start calling it DownloadURL instead of "the Download URL"